R/flatten.R

```diff
@@ -45,29 +99,52 @@ cdm_flatten_to_tbl <- function(dm, start, join = left_join) {
       name = names(dfs[["order"]]),
       pred = names(V(g))[ unclass(dfs[["father"]])[name] ]
     ) %>%
-    filter(!is.na(name))
+    filter(!is.na(name), name %in% c(start, list_of_pts))
```

Comment:
`%in%` implies `!is.na`:

```suggestion
    filter(name %in% c(start, list_of_pts))
```